test/_init_tests.js

```diff
@@ -55,25 +55,26 @@ class TestConfig {
      * @type {!Array<function(!TestSuite)>}
      */
     this.configTasks = [];
+    this.platform_ = platformFor(window);
```

Comment:
lol  designed a fashionable T-Shirt for you:

![lztrksjlqhq](https://cloud.githubusercontent.com/assets/2099009/17419475/71676fe6-5a52-11e6-8c07-02632dfc9080.png)